confluent_platform/datadog_checks/confluent_platform/data/metrics.yaml

```diff
@@ -1187,10 +1183,6 @@ jmx_metrics:
           # The average number of requests per second that resulted in HTTP error responses
           alias: confluent.$domain.jersey.$attribute
           metric_type: gauge
-        partitions.list+v2.request-error-rate:
```

Comment:
Dupe of https://github.com/DataDog/integrations-core/pull/11095/files#diff-827f19bd523f493e7fbe89480ef616a47c3626340f66fe71a74b8f890d24782aR1170